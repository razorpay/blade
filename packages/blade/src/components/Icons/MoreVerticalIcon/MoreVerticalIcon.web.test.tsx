import MoreVerticalIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<MoreVerticalIcon />', () => {
  it('should render MoreVerticalIcon', () => {
    const { container } = renderWithTheme(
      <MoreVerticalIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
