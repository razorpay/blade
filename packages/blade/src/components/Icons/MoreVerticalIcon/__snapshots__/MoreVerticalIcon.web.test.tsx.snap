// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoreVerticalIcon /> should render MoreVerticalIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="Svgweb__StyledSvg-vcmjs8-0"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      clip-rule="evenodd"
      d="M9 12C9 10.3431 10.3431 9 12 9C13.6569 9 15 10.3431 15 12C15 13.6569 13.6569 15 12 15C10.3431 15 9 13.6569 9 12Z"
      fill="hsla(216, 27%, 36%, 1)"
      fill-rule="evenodd"
    />
    <path
      clip-rule="evenodd"
      d="M9 4C9 2.34315 10.3431 1 12 1C13.6569 1 15 2.34315 15 4C15 5.65685 13.6569 7 12 7C10.3431 7 9 5.65685 9 4Z"
      fill="hsla(216, 27%, 36%, 1)"
      fill-rule="evenodd"
    />
    <path
      clip-rule="evenodd"
      d="M9 20C9 18.3431 10.3431 17 12 17C13.6569 17 15 18.3431 15 20C15 21.6569 13.6569 23 12 23C10.3431 23 9 21.6569 9 20Z"
      fill="hsla(216, 27%, 36%, 1)"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
